.github/workflows/ci.yml

```diff
@@ -61,7 +61,26 @@ jobs:
       matrix:
         profile: [test_with_library, test_global_fdr, test_no_align, test_no_merge, test_with_library_with_decoys, test_with_library_without_irts, test_without_library_with_irts]
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out pipeline code
+        uses: actions/checkout@v2
+
+      - name: Check if Dockerfile or Conda environment changed
+        uses: technote-space/get-diff-action@v1
+        with:
+          PREFIX_FILTER: |
+            Dockerfile
+            environment.yml
+
+      - name: Build new docker image
+        if: env.GIT_DIFF
+        run: docker build --no-cache . -t nfcore/diaproteomics:1.0.0
+
+      - name: Pull docker image
+        if: ${{ !env.GIT_DIFF }}
+        run: |
+          docker pull nfcore/diaproteomics:dev
+          docker tag nfcore/diaproteomics:dev nfcore/diaproteomics:1.0.0
```

Comment:
Check line 92: there `sarek` container is pulled instead of `diaproteomics`

Reply:
ha ok thx! I'll fix it.

Reply:
done